tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -191,16 +154,33 @@ var _ = Describe("QinQ", Ordered, Label(tsparams.LabelQinQTestCases), ContinueOn
 			if sriovDeviceID != intelDeviceIDE810 {
 				Skip(fmt.Sprintf("The NIC %s does not support 802.1AD", sriovDeviceID))
 			}
+			By("Verify Sriov Polices are created.")
+			policyList, err := sriov.ListPolicy(APIClient, NetConfig.SriovOperatorNamespace)
+			Expect(err).ToNot(HaveOccurred(), "Unable to pull Policy list")
+
+			if len(policyList) != 2 {
```

Comment:
it's really weird check. If we have some leftover policy or default?

Reply:
What was the motivation of this change?

Reply:
The last Context "nmstate" deletes all sriovpolicies and sriovnetworks.  If this Context is not the last to run then the Context 802.1AD and 802.1Q will fail. So its a safe guard.

Reply:
Add policy  creation only to Context 802.1AD and 802.1Q and delete it from the general beforeall. 
Currently it looks very confusing and massive:
```
General beforeall: Create SR-IOV configuration
"nmstate" beforeall: Remove SR-IOV configuration
```

Reply:
removed